packages/concerto-cli/index.js

```diff
@@ -54,6 +54,28 @@ require('yargs')
                 console.log(err.message + ' ' + err);
             });
     })
+    .command('saveExternal', 'save local copies of external model dependencies', (yargs) => {
+        yargs.option('ctoFiles', {
+            describe: 'array of local CTO files',
+            type: 'string',
+            array: true,
+            default: '.'
+        });
+        yargs.option('outputDirectory', {
```

Comment:
Some of our other tools use `--out` (not always for directories though) so maybe we could try and align?